docs/source/development.rst

```diff
@@ -61,25 +68,32 @@ Documentation
 Building the docs with Sphinx
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-The documentation (in reStructuredText format, .rst) is in docs/source/.
+The documentation (in reStructuredText format, .rst) is in ``docs/source/``,
+``index.rst`` is the starting point there.
 
-To build the html version of it, you need to have Sphinx installed.
+To build the docs, you need to have Sphinx_ installed and run:
 
-Now run::
+::
 
   cd docs/
   make html
 
-Then point a web browser at docs/build/html/index.html.
+Then point a web browser at ``docs/build/html/index.html``.
 
 The website is updated automatically by ReadTheDocs through GitHub web hooks on the
 main repository.
 
+.. _Sphinx: https://www.sphinx-doc.org/
+
```

Comment:
@rexfue Do we need to add a link to https://docutils.sourceforge.io/rst.html or can it be found easily enough on the sphinx homepage?

Reply:
OK, I found it. Others should find it too - so no link necessary.